contents/navigation/navbars/double-navbar/index.tsx

```diff
@@ -0,0 +1,136 @@
+import {
+  BarChart3,
+  Calendar,
+  Fingerprint,
+  Gauge,
+  Home,
+  Settings,
+  User,
```

Comment:
I think `<BarChart3Icon />` might be preferable than `<BarChart3 />` and same for the others. What do you think?

Reply:
OK, I will fix it.
Should the other icons be modified in the same way?

Reply:
Yes, please!

Reply:
Thanks!
Let's create an issue because I think the same problem exists in other components.

Reply:
Sounds good, thanks!

Reply:
I fixed it!

Reply:
I created issue.

https://github.com/yamada-ui/yamada-components/issues/280